docs/paying-for-lit/overview.md

```diff
@@ -0,0 +1,60 @@
+
+In order to use the paid pre-production ([Datil-test](../connecting-to-a-lit-network/testnets#datil-test)) and production ([Datil](../connecting-to-a-lit-network/mainnets#datil)) Lit networks, you must reserve capacity on the network. This is done using by minting Capacity Credits, and by making use of the Lit Relayer and/or the Payment Delegation Database.
+
+## Overview of Payment Methods
+
+### Capacity Credits
+
+Capacity Credits are NFT tokens that represent reserved computational capacity on the Lit network. When minting a credit, you specify and pay to reserve a specific amount of requests per second over a period of time. You'll then use these credits when making requests to the Lit network to perform actions such as decryption, executing Lit Actions, and using PKPs.
+
+For a deep dive into Capacity Credits, including minting and usage details, checkout the in-depth [documentation](./capacity-credit-intro.md).
+
+### Lit Relayer
+
+The Lit Relayer is an [open-source service](https://github.com/LIT-Protocol/relay-server) currently hosted by Lit to facilitate onboarding into the Lit ecosystem. It helps reduce initial costs by covering or subsidizing certain interactions with the Lit network, such as minting PKPs and paying for network requests to perform actions like decryption, executing Lit Actions, and using PKPs.
+
+While the Relayer eases the onboarding process, it's important to note that its availability is not guaranteed. Users may experience rate limiting and/or congestion due to its shared nature.
+
+As you progress with your Lit integration, we recommend implementing Capacity Credits and/or the Payment Delegation Database in your application. These solutions offer more reliable and scalable options for managing your long-term usage of the Lit network.
+
+For a deep dive into the Relayer, including its usage and offered services, checkout the in-depth [documentation](./lit-relayer.md).
+
+### Payment Delegation Database
+
+The Payment Delegation Database is a service provided by the Lit Relayer that streamlines payment management for your users. This service offers an alternative to relying on the Lit Relayer's payment subsidies or minting individual Capacity Credits for each of your users. With the Payment Delegation Database, you can establish a Payer Wallet and designate your users as Payees.
+
+The Payer Wallet acts as a central account that manages payment for Lit network usage on behalf of your users (Payees). Payees inherit the reserved capacity of the Payer Wallet, which automatically receives a minted capacity credit when registered with the service.
+
+For a deep dive into the Payment Delegation Database, including how to register a Payer Wallet and add users as Payees, checkout the in-depth [documentation](./payment-delegation-db.md).
+
+### Choosing the Right Payment Method
```

Comment:
@spacesailor24 wondering how we might word this given our discussion today about the relayer and payment delegation DB not necessarily being maintained long-term. Capacity credits are eventually going away too. So maybe we can suggest the relayer + payment delegation DB for rapid prototyping for now, and moving to self-hosted versions of those or direct capacity credit integration for production?

Reply:
Probably helpful to also tell people how they ought to go about self-hosting these services to get production-ready (where the code lives, things to consider, etc.) somewhere. Maybe in the repos for those tools themselves, with a link to them here?

Reply:
After the discussion on the strategy call, it seems like we'll be supporting these services for users, however I think it's still unclear to what capacity i.e. will this support user's production apps, or is this still for prototyping